modules/virt-creating-nad-localnet-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-nad-localnet-console_{context}"]
+= Creating a NAD for localnet topology using the web console
+
+You can create a network attachment definition (NAD) to connect workloads to a physical network by using the {product-title} web console.
+
+.Prerequisites
+* Use nmstate to configure a localnet identifier.
```

Comment:
```suggestion
* Use nmstate to configure the localnet to OVS bridge mappings.
```

And probably in the additional resources explicitly list `Configuration for an OVN-Kubernetes additional network mapping` - i.e. https://70940--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-additional-network#configuring-additional-network_configuration-additional-network-interface to help the user out.